source/databases-collections.txt

```diff
@@ -117,33 +117,32 @@ Create a New Database
 When you create a new database, you must populate it with an
 initial collection.
 
-To create a new database:
-
-1. Hover over the connection for the deployment where you want your
+#. Hover over the connection for the deployment where you want your
    database to exist.
 
 #. Click the :icon-fa5:`plus` icon that appears.
+   |vsce| opens a new tab with ``const`` variables for database and 
+   collection names.
 
-#. In the prompt, enter a name for your new database.
-
-#. Press the enter key.
-
-#. Enter a name for the first collection in your new database.
+#. Update the ``database`` and ``collection`` variables with 
+   the names for your database and collection.
 
-#. Press the enter key.
+#. Click the white :icon-fa5:`play` button to execute the script.
```

Comment:
We might want to remove the color reference here and below where we mention the button is `white` as it depends on the user's theme:
| dark | light |
| - | - |
| <img width="239" alt="Screenshot 2023-07-24 at 1 41 11 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/1791149/bba1602b-63f9-434e-8707-002a10014676"> | <img width="164" alt="Screenshot 2023-07-24 at 1 41 47 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/1791149/474589d7-79fb-4077-bad3-c2e5aa03cb42"> |

Instead how about we mention the button is at the top right of the editor window?
```suggestion
#. Click the :icon-fa5:`play` button located at the top right of the playground editor tab to execute the script.
```